### ISSUE TITLE
ports/webassembly: import into node fails

### ISSUE THREAD

**Issue by hoihu** [micropython/micropython#14363]
### Checks

- [X] I agree to follow the MicroPython [Code of Conduct](https://github.com/micropython/micropython/blob/master/CODEOFCONDUCT.md) to ensure a safe and respectful space for everyone.

- [X] I've searched for [existing issues](https://github.com/micropython/micropython/issues) matching this bug, and didn't find any.

### Port, board and/or hardware

macOS, Node V20.11, Emscripten Port

### MicroPython version

`MicroPython v1.23.0-preview.324.g6634fea23.dirty on 2024-04-23; JS with Emscripten`

### Reproduction

```
fischmar@MACNP7L4XQC6C build-pyscript % node                
Welcome to Node.js v20.11.1.
Type ".help" for more information.
> const mp_mjs = await import("./micropython.mjs");
Uncaught:
TypeError [ERR_INVALID_ARG_TYPE]: The "paths[0]" argument must be of type string. Received undefined
    at validateString (node:internal/validators:162:11)
    at Module.resolve (node:path:1101:7) {
  code: 'ERR_INVALID_ARG_TYPE'
}
```

### Expected behaviour

Import in node does not cause a fault.

Issuing `node micropython.mjs` correctly gives me the repl, so the `mjs` file seems to be ok.

Or maybe I'm missing something here? I was following the readme, section here:
```javascript
const mp_mjs = await import("micropython.mjs");
const mp = await mp_mjs.loadMicroPython();

mp.runPython("print('hello world')");
```

### Observed behaviour

see above

### Additional Information

No, I've provided everything above.

**Comment by dpgeorge** [comment]
Thanks for the report.  Should be fixed by #14411.

**Comment by hoihu** [comment]
Thank you for the quick fix 👍

**Comment by dpgeorge** [comment]
Fixed by 9681a66c6b79a5f4721c74b756256336c47c9879

**Comment by hoihu** [comment]
confirmed, that has fixed it, thanks again.
